From 9a1e9120a340e634732af2ee1512e8b010a9a934 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sun, 30 Apr 2023 17:26:23 +0300
Subject: [PATCH 4/7] uboot: Fix sunxi-specific definition

Semi-cosmetic change. Should not change anything for the firmware.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 platform/uboot/platform_sunxi.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/platform/uboot/platform_sunxi.h b/platform/uboot/platform_sunxi.h
index 14415c4..09771f9 100644
--- a/platform/uboot/platform_sunxi.h
+++ b/platform/uboot/platform_sunxi.h
@@ -1,10 +1,10 @@
 #pragma once
 
 #define PLATFORM_SETUP_ENV() \
- setenv dtbaddr 0x5fa00000;
- setenv loadaddr 0x50008000;
- setenv vloadaddr 0x53008000;
- setenv dtboaddr 0x52008000;
+ setenv dtbaddr 0x5fa00000;  \
+ setenv loadaddr 0x50008000; \
+ setenv vloadaddr 0x53008000;\
+ setenv dtboaddr 0x52008000; \
 
 #define PLATFORM_HANDLE_FDT() \
  adtimg get dt --id=\$main_fdt_id dtb_start dtb_size main_fdt_index && \
-- 
2.37.2

